scripts/test.sh

```diff
@@ -63,18 +63,17 @@ else
 fi
 
 if [ "$SOLC_NIGHTLY" = true ]; then
-  echo "Downloading solc nightly"
-  wget -q https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/soljson-nightly.js -O /tmp/soljson.js && find . -name soljson.js -exec cp /tmp/soljson.js {} \;
+  docker pull ethereum/solc:nightly
```

Comment:
Is Docker available by default on Travis, or do we have to install it somehow?

Reply:
It's available by default. Note that the job in this PR passed. :slightly_smiling_face: